From bd2eeaddfc1f12f87184d870cc9a1adde1cf0b10 Mon Sep 17 00:00:00 2001
From: Mark Asselstine <mark.asselstine@windriver.com>
Date: Mon, 24 Apr 2017 13:08:48 -0400
Subject: [PATCH] criu/Makefile.config: explicitly remove selinux support

Upstream-Status: Inappropriate [disable feature]

It shows warning when build crius if libselinux has been built already:

  WARNING: QA Issue: criu rdepends on libselinux, but it isn't a build dependency? [build-deps]

Apply this patch to disable selinux support when 'selinux' is not in PACKAGECONF.

Signed-off-by: Kai Kang <kai.kang@windriver.com>
[MA: Context updated to apply against criu v2.5]
Signed-off-by: Mark Asselstine <mark.asselstine@windriver.com>
---
 criu/Makefile.config | 5 -----
 1 file changed, 5 deletions(-)

diff --git a/criu/Makefile.config b/criu/Makefile.config
index f531b3b..37216f8 100644
--- a/criu/Makefile.config
+++ b/criu/Makefile.config
@@ -7,11 +7,6 @@ ifeq ($(call try-cc,$(FEATURE_TEST_LIBBSD_DEV),-lbsd),true)
         FEATURE_DEFINES	+= -DCONFIG_HAS_LIBBSD
 endif
 
-ifeq ($(call pkg-config-check,libselinux),y)
-        LIBS		+= -lselinux
-        FEATURE_DEFINES	+= -DCONFIG_HAS_SELINUX
-endif
-
 export DEFINES += $(FEATURE_DEFINES)
 export CFLAGS += $(FEATURE_DEFINES)
 
-- 
2.7.4

